packages/manageiq-ansible-venv/container-assets/build.sh

```diff
@@ -7,20 +7,20 @@ rpm_name=$PRODUCT_NAME-ansible-venv
 rpm_path=/root/rpms
 
 echo "*** Installing python system packages"
-pip3.8 install virtualenv psutil==5.6.6
+pip3 install virtualenv psutil==5.6.6
```

Comment:
Same here - this was intentionally locked to a specific version, so I'm thinking this should be pip3.9

Reply:
Done